src/main/java/uk/gov/hmcts/reform/prl/services/ServiceOfApplicationPostService.java

```diff
@@ -119,111 +119,130 @@ public List<Document> getStaticDocs(String auth, String caseType, CaseData caseD
+        if (documentLanguage.isGenEng()) {
+            files.add(new InMemoryMultipartFile(
+                SOA_MULTIPART_FILE,
+                PRIVACY_DOCUMENT_FILENAME,
+                APPLICATION_PDF_VALUE,
+                DocumentUtils.readBytes(URL_STRING + ENG_STATIC_DOCS_PATH + PRIVACY_DOCUMENT_FILENAME)
+            ));
+        }
+        if (documentLanguage.isGenWelsh()) {
+            files.add(new InMemoryMultipartFile(
+                SOA_MULTIPART_FILE,
+                PRIVACY_DOCUMENT_FILENAME_WELSH,
+                APPLICATION_PDF_VALUE,
+                DocumentUtils.readBytes(URL_STRING + ENG_STATIC_DOCS_PATH + PRIVACY_DOCUMENT_FILENAME_WELSH)
+            ));
+        }
+        if (documentLanguage.isGenEng()) {
+            files.add(new InMemoryMultipartFile(
+                SOA_MULTIPART_FILE,
+                SOA_FL415_FILENAME,
+                APPLICATION_PDF_VALUE,
+                DocumentUtils.readBytes(URL_STRING + ENG_STATIC_DOCS_PATH + SOA_FL415_FILENAME)
+            ));
+        }
+        return caseDocumentClient.uploadDocuments(
+            auth,
+            authTokenGenerator.generate(),
+            PrlAppsConstants.CASE_TYPE,
+            PrlAppsConstants.JURISDICTION,
+            files
+        );
+    }
+
+    private UploadResponse getC100UploadResponse(String auth, DocumentLanguage documentLanguage) {
+        List<MultipartFile> files = new ArrayList<>();
+        if (documentLanguage.isGenEng()) {
+            files.add(new InMemoryMultipartFile(
+                SOA_MULTIPART_FILE,
+                PRIVACY_DOCUMENT_FILENAME,
+                APPLICATION_PDF_VALUE,
+                DocumentUtils.readBytes(URL_STRING + ENG_STATIC_DOCS_PATH + PRIVACY_DOCUMENT_FILENAME)
+            ));
+        }
+        if (documentLanguage.isGenWelsh()) {
+            files.add(new InMemoryMultipartFile(
+                SOA_MULTIPART_FILE,
+                PRIVACY_DOCUMENT_FILENAME_WELSH,
+                APPLICATION_PDF_VALUE,
+                DocumentUtils.readBytes(URL_STRING + ENG_STATIC_DOCS_PATH + PRIVACY_DOCUMENT_FILENAME_WELSH)
+            ));
+        }
+        //PRL-5360 - Remove mediation voucher & add new President note
+        if (documentLanguage.isGenEng()) {
+            files.add(new InMemoryMultipartFile(
+                SOA_MULTIPART_FILE,
+                SOA_FAMILY_PRESIDENTS_NOTE,
+                APPLICATION_PDF_VALUE,
+                DocumentUtils.readBytes(URL_STRING + ENG_STATIC_DOCS_PATH + SOA_FAMILY_PRESIDENTS_NOTE)
+            ));
+        }
+        if (documentLanguage.isGenWelsh()) {
+            files.add(new InMemoryMultipartFile(
+                SOA_MULTIPART_FILE,
+                SOA_FAMILY_PRESIDENTS_NOTE_WELSH,
+                APPLICATION_PDF_VALUE,
+                DocumentUtils.readBytes(URL_STRING + ENG_STATIC_DOCS_PATH + SOA_FAMILY_PRESIDENTS_NOTE_WELSH)
+            ));
+        }
+        files.addAll(
```

Comment:
Follow the previous static docs logc to add to the list, if welsh is not available then add the block and leave it empty with a comment to say "when welsh doc is available it can be added."

Reply:
Ignore this comment as we dont have welsh version of the doc, and in a scenario where welsh language is choose and the welsh doc is not available then the pack witll not have both english and welsh, which is not the expected behaviour.